r/tools/nixlibs.R

```diff
@@ -133,16 +125,16 @@ identify_binary <- function(lib = Sys.getenv("LIBARROW_BINARY"), info = distro()
 
   if (identical(lib, "false")) {
     # Do not download a binary
-    NULL
+    lib <- NULL
   } else if (!identical(lib, "true")) {
     # Env var provided an os-version to use, to override our logic.
     # We don't validate that this exists. If it doesn't, the download will fail
     # and the build will fall back to building from source
-    lib
```

Comment:
Is this intended to be a no-op? It's a little bit weird IMHO to have a branch with no operative code and only a comment

Reply:
Yes it's intentional as no change to `lib` is needed. The alternatives are `lib <- lib` or using a different var for the return. Both are :shrug: though maybe `lib <- lib` is a bit more explicit version of a noop :)

Reply:
There are other ways to avoid this: You could also not have that branch (and possibly edit the conditions branch below), no? Or keep the return-the-last-value logic that was there?